src/main/java/com/pnudev/springonlinedelivery/SpringOnlineDeliveryApplication.java

```diff
@@ -1,13 +1,25 @@
 package com.pnudev.springonlinedelivery;
 
+import java.util.List;
+
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
 
 @SpringBootApplication
+@RestController
```

Comment:
First of all, you are violating the single-responsibility principle by making your main class also a controller.
Second of all, I cannot exactly understand why do you need this kind of "test" endpoint now. Just delete it for now and replace it with actual endpoint when login/signup is implemented

Reply:
I did it to see for myself if the server was working. Removed.